include/blmc_robots/real_finger.hpp

```diff
@@ -130,19 +216,7 @@ class RealFingerDriver : public robot_interfaces::RobotDriver<
         }
     }
 
-protected:
-    Eigen::Vector3d max_angles_;
-
-    static Motors create_motors(const MotorBoards &motor_boards)
-    {
-        // set up motors -------------------------------------------------------
-        Motors motors;
-        motors[0] = std::make_shared<blmc_drivers::Motor>(motor_boards[0], 0);
-        motors[1] = std::make_shared<blmc_drivers::Motor>(motor_boards[0], 1);
-        motors[2] = std::make_shared<blmc_drivers::Motor>(motor_boards[1], 0);
-
-        return motors;
-    }
+    virtual CalibrationParameters get_calibration_parameters() = 0;
```

Comment:
unless there is a strong reason, i would not implement this in this way. i would just pass the calibration_parameters as an argument in the constructor, rather than having this virtual function which needs to be implemented by the child class.

the reasons being that this seems more logical design to me, and also that this way NJointBlmcRobotDriver is not a virtual class, and we can actually create an instance without deriving.

Reply:
done